Sources/GDBRemote/SessionBase.cpp

```diff
@@ -139,6 +139,12 @@ bool SessionBase::sendACK() { return _channel->send("+", 1) == 1; }
 
 bool SessionBase::sendNAK() { return _channel->send("-", 1) == 1; }
 
+void SessionBase::addPacketSeparator(std::ostringstream &ss) {
```

Comment:
You should have a comment in this function to explain what we're trying to achieve.